src/vs/workbench/browser/parts/editor/stringEditor.ts

```diff
@@ -156,18 +149,12 @@ export class StringEditor extends BaseTextEditor {
 		let options = super.getCodeEditorOptions();
 
 		let input = this.getInput();
-		let isLog = input instanceof LogEditorInput;
 		let isUntitled = input instanceof UntitledEditorInput;
 
 		options.readOnly = !isUntitled; 				// all string editors are readonly except for the untitled one
 
-		if (isLog) {
-			options.wrappingColumn = 0;					// all log editors wrap
-			options.lineNumbers = false;				// all log editors hide line numbers
-		} else {
-			options.wrappingColumn = this.defaultWrappingColumn; 	// otherwise make sure to restore the defaults
-			options.lineNumbers = this.defaultLineNumbers; 			// otherwise make sure to restore the defaults
-		}
+		options.wrappingColumn = this.defaultWrappingColumn; 	// otherwise make sure to restore the defaults
```

Comment:
@isidorn suggest to remove the concept of defaultWrappingColumn and defaultLineNumbers because it was only needed for the outputEditorInput and this now moved nicely into the outputPanel